src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,41 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectTodo: Todo;
+  setSelectTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectTodo, setSelectTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const userData = async () => {
```

Comment:
Consider renaming the `userData` function to `fetchUserData` for better clarity and consistency with naming conventions.

Reply:
> Consider renaming the `userData` function to `fetchUserData` for better clarity and consistency with naming conventions.

Good point